src/test/java/com/artipie/RepositoriesFromStorageTest.java

```diff
@@ -85,7 +85,7 @@ void throwsExceptionWhenConfigYamlAbsent() {
             this::repoConfig
         );
         MatcherAssert.assertThat(
-            result.getCause(),
+            result.getCause().getCause(),
```

Comment:
This change looks not related to PR modification, isn't it?

Reply:
@g4s8 now `ValueNotFoundException` is wrapped with `java.util.concurrent.ExecutionException` therefore this test was changed